infrastructure/local/docker-compose.yml

```diff
@@ -1,6 +1,15 @@
 version: "3.9"
 services:
 
+  ui:
+    build:
+      context: ../../ui/
+      dockerfile: Dockerfile
+    ports:
+      - "8080:80"
+    environment:
+      - API=https://api-staging/polygonid/com/v1
```

Comment:
Should be this `localhost:3001`?

Reply:
Hi, it should be something like `http://backend:3002`. backend service is not merged yet, but after merging it we can call it with the service name (instead of the url)

Reply:
Yeah.. you are right.. Let's merge https://github.com/0xPolygonID/sh-id-platform/pull/190 before and i will change it

Reply:
done